pages/docs/concepts/asyncapi document/url-variable.md

```diff
@@ -0,0 +1,108 @@
+---
+title: URL Variables
+weight: 275
+---
+
+[Event-driven architecture](../../tutorials/getting-started/event-driven-architectures) benefits from a defined base URL and rules for URL variables. You can easily manage multiple API endpoints, handling various server configurations and environments.
```

Comment:
Technically correct but suggested by Lukasz to use the relative links as follows:

```suggestion
[Event-driven architecture](/docs/tutorials/getting-started/event-driven-architectures) benefits from a defined base URL and rules for URL variables. You can easily manage multiple API endpoints, handling various server configurations and environments.
```